src/common/pick_address.h

```diff
@@ -20,6 +20,12 @@ class entity_addrvec_t;
 #define CEPH_PICK_ADDRESS_PREFER_IPV4 0x40
 #define CEPH_PICK_ADDRESS_DEFAULT_MON_PORTS  0x80
 
+enum IfaceType {
+  IFACE_DEFAULT = 0,
+  IFACE_PHY_PORT = 1,
+  IFACE_BOND_PORT = 2
+};
+
```

Comment:
nit: this doesn't need to be in the header

Reply:
@rosinL could you address this comment?

Reply:
ok,I will remove the enum to pick_address.cc